plan/logical_plan_builder.go

```diff
@@ -908,8 +909,39 @@ func (b *planBuilder) buildTableDual() LogicalPlan {
 }
 
 func (b *planBuilder) getTableStats(table *model.TableInfo) *statistics.Table {
-	// TODO: Currently we always return a pseudo table for good performance. We will use a cache in future.
-	return statistics.PseudoTable(table)
+	tbl := statistics.GetStatisticsTableCache(table)
+	if tbl != nil {
+		return tbl
+	}
+	txn := b.ctx.Txn()
+	if txn == nil {
+		return statistics.PseudoTable(table)
+	}
+	m := meta.NewMeta(txn)
```

Comment:
I tried, but it will introduce circle dependency, because `meta/meta.go` imports `statistics` package. And if stats table is nil, I will save a pseduo one in cache, thus we will not get it from kv every time.